src/mgr/OSDPerfMetricTypes.cc

```diff
@@ -2,7 +2,7 @@
 // vim: ts=8 sw=2 smarttab
 
 #include "mgr/OSDPerfMetricTypes.h"
-
+#include "common/Formatter.h"
```

Comment:
Our includes should appear after any standard ones. I know it's non-new but it would be really nice if we can fix that.